sumpy/e2e.py

```diff
@@ -600,17 +583,22 @@ def get_kernel(self, result_dtype):
                 name=self.name,
                 assumptions="ntranslation_classes>=1",
                 default_offset=lp.auto,
-                fixed_parameters=dict(dim=self.dim),
+                fixed_parameters=dict(
+                    dim=self.dim,
+                    m2l_translation_classes_dependent_ndata=(
+                        m2l_translation_classes_dependent_ndata)),
                 lang_version=MOST_RECENT_LANGUAGE_VERSION
                 )
 
         for knl in [self.src_expansion.kernel, self.tgt_expansion.kernel]:
```

Comment:
```suggestion
        for expn_knl in [self.src_expansion.kernel, self.tgt_expansion.kernel]:
```
(for clarity?)

Reply:
Done in 9cc214e7e7827a54784b9eadf8a275b70deb64b5